src/main/resources/db/migration/v0.31.0.4__agm_phenotype_annotation.sql

```diff
@@ -0,0 +1,128 @@
+CREATE INDEX genephenotypeannotation_phenotypeannotationsubject_index ON genephenotypeannotation USING btree (phenotypeannotationsubject_id);
+CREATE INDEX genephenotypeannotation_sgdstrainbackground_index ON genephenotypeannotation USING btree (sgdstrainbackground_id);
+
+-- Add relation vocabulary with 'has_phenotype' term
+
+INSERT INTO vocabulary (id, name, vocabularylabel) VALUES (nextval('vocabulary_seq'), 'Phenotype Relation', 'phenotype_relation');
+INSERT INTO vocabularyterm (id, name, vocabulary_id) SELECT nextval('vocabularyterm_seq'), 'has_phenotype', id FROM vocabulary WHERE vocabularylabel = 'phenotype_relation';
+
+-- Create bulk loads
+
+INSERT INTO bulkloadgroup (id, name) VALUES (nextval('bulkloadgroup_seq'), 'Phenotype Bulk Loads');
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+	SELECT nextval('bulkload_seq'), 'PHENOTYPE', 'FB Phenotype Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'Phenotype Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+	SELECT nextval('bulkload_seq'), 'PHENOTYPE', 'HUMAN Phenotype Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'Phenotype Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+	SELECT nextval('bulkload_seq'), 'PHENOTYPE', 'MGI Phenotype Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'Phenotype Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+	SELECT nextval('bulkload_seq'), 'PHENOTYPE', 'RGD Phenotype Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'Phenotype Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+	SELECT nextval('bulkload_seq'), 'PHENOTYPE', 'SGD Phenotype Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'Phenotype Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+	SELECT nextval('bulkload_seq'), 'PHENOTYPE', 'WB Phenotype Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'Phenotype Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+	SELECT nextval('bulkload_seq'), 'PHENOTYPE', 'XBXL Phenotype Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'Phenotype Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+	SELECT nextval('bulkload_seq'), 'PHENOTYPE', 'XBXT Phenotype Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'Phenotype Bulk Loads';
+INSERT INTO bulkload (id, backendbulkloadtype, name, bulkloadstatus, group_id)
+	SELECT nextval('bulkload_seq'), 'PHENOTYPE', 'ZFIN Phenotype Load', 'STOPPED', id FROM bulkloadgroup WHERE name = 'Phenotype Bulk Loads';
+INSERT INTO bulkscheduledload (id, cronschedule, scheduleactive)
+	SELECT id, '0 0 22 ? * SUN-THU', true FROM bulkload WHERE backendbulkloadtype = 'PHENOTYPE';
```

Comment:
Right now the scheduleactive is set to false on the scheduled loads... will create a migration in the future to set them to true.

Reply:
Have set to false